tests/model_fields/test_uuid.py

```diff
@@ -35,6 +35,14 @@ def test_null_handling(self):
         self.assertEqual(loaded.field, None)
 
 
+class TestMigrations(TestCase):
```

Comment:
Yes, there is some debate about whether to use `TestCase` anyway to prevent someone from inadvertently adding a database test and forgetting to change this.